deploy/custom_resources/cognito_config/cognito_urls.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+log = logging.getLogger(__name__)
+
+
+def setup_cognito(
+    region,
+    envname,
+    custom_domain='False',
+):
+    ssm = boto3.client('ssm', region_name=region)
+    user_pool_id = ssm.get_parameter(Name=f'/dataall/{envname}/cognito/userpool')['Parameter']['Value']
+    log.info(f'Cognito Pool ID: {user_pool_id}')
+    app_client = ssm.get_parameter(Name=f'/dataall/{envname}/cognito/appclient')['Parameter']['Value']
+
+    if custom_domain == 'False':
```

Comment:
We are removing the condition of internet_facing because the cloudfront stack is only deployed when it is internet facing